src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int FIGURE_PROPERTY = 10;
+    public static final int FIGURE_PROPERTY_DEFAULT = 10;
+    public static final String COLOR_DEFAULT = "WHITE";
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    private String getColorRandom() {

```

Comment:
According to the codestyle, public methods go first, then private ones.

Reply:
Ok, thanks!